index.html

```diff
@@ -217,6 +217,41 @@ <h2>Languages</h2>
           </div>
         </div>
       </section>
+      <section class="contact-form">
+        <h3>Contact me</h3>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] Kindly note that you're required to create a formspree account and get an ID to replace the `{form_id}` in your markup. Kindly [watch](https://www.youtube.com/watch?v=sVPlb5C3oQ8) how it is done. 🌔

- And the value of your `method` should be "POST" in uppercase.
<img width="166" alt="wrong" src="https://user-images.githubusercontent.com/41163571/228314330-f98398fa-1c6e-491e-9663-e455a6b1f4aa.png">